doc/classes/BakedLightmap.xml

```diff
@@ -32,8 +34,10 @@
 			Grid subdivision size for lightmapper calculation. The default value will work for most cases. Increase for better lighting on small details or if your scene is very large.
 		</member>
 		<member name="bake_default_texels_per_unit" type="float" setter="set_bake_default_texels_per_unit" getter="get_bake_default_texels_per_unit" default="20.0">
+			If a [code]lightmap_size_hint[/code] isn't specified on a [code]Mesh[/code], the lightmap baker will dynamically set the lightmap size using this value. This value is measured in texels per world unit. The maximum lightmap texture size is 4096x4096.
```

Comment:
You can use `[member Mesh.lightmap_size_hint]` to have a direct reference to that property's docs, and `[Mesh]` to link the Mesh docs.

Reply:
Thanks for the tip Remi, this should be updated now :)